packages/discussions/src/utils/channel-permission.ts

```diff
@@ -164,21 +176,22 @@ export class ChannelPermission {
     });
   }
 
-  /**
-   * canCreateChannelHelpers
-   */
   private userCanAddAnonymousToAcl(user: IDiscussionsUser) {
     if (!this.permissionsByCategory[AclCategory.ANONYMOUS_USER]) {
       return true;
     }
-    return isOrgAdmin(user);
+    return (
+      isOrgAdmin(user) || userHasPrivilege(user, "portal:admin:shareToPublic")
```

Comment:
the `user is in the channel org` check is not necessary here, since these functions are used on channel create only